src/Silex/ApplicationTrait.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Silex;
+
+use Composer\Semver\VersionParser;
+use Composer\InstalledVersions;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+
+if (InstalledVersions::satisfies(new VersionParser, 'symfony/http-kernel', '^6.0.0')) {
```

Comment:
Do we still support `spryker/silexphp`? @stereomon

Reply:
We shouldn't

Reply:
@geega does this package update block us from making the suite-nonsplit Travis green?

Reply:
@asmarovydlo  Yes. Only for fixing Travis test issues.